docs/en/integrations/data-ingestion/s3/gcs-merge-tree.md

```diff
@@ -76,19 +70,19 @@ The example configuration highlighted below enables a 10Gi memory cache for the
                 <metadata_path>/var/lib/clickhouse/disks/gcs/</metadata_path>
             </gcs>
 	    <!--highlight-start-->
-	    <cache>
+	    <gcs_cache>
                 <type>cache</type>
                 <disk>gcs</disk>
                 <path>/var/lib/clickhouse/disks/gcs_cache/</path>
                 <max_size>10Gi</max_size>
-            </cache>
+            </gcs_cache>
 	    <!--highlight-end-->
         </disks>
         <policies>
             <gcs_main>
                 <volumes>
                     <main>
-                        <disk>gcs</disk>
+                        <disk>gcs_cache</disk>
```

Comment:
This place was wrong - it did not enable cache as stated “The example configuration highlighted below enables a 10Gi memory cache for the disk `gcs`.” . It made a user wonder why it does not work: https://clickhousedb.slack.com/archives/CU478UEQZ/p1675420392261419?thread_ts=1675350475.829829&cid=CU478UEQZ.

Reply:
@kssenii I don't think gcs_cache belongs in the policies.

Reply:
what do you mean? Otherwise cache will not be enabled.